src/application/extension-popup/views/home/components/section-tile.tsx

```diff
@@ -0,0 +1,65 @@
+import { classes, Icon } from 'shared/ui';
+
+interface SectionTileIconProperties {
+  iconSrc: string;
+  iconAlt: string;
+  className?: string;
+}
+interface SectionTileProperties {
+  title: string;
+  subtitle: string;
+  icons: SectionTileIconProperties[];
+  href?: string;
+  disabled?: boolean;
+  iconContainerClassName?: string;
+}
+
+export const SectionTile = ({
+  icons,
+  title,
+  subtitle,
+  href,
+  disabled,
+  iconContainerClassName,
+}: SectionTileProperties) => {
+  return (
+    <a
+      href={href}
+      target={href ? '_blank' : undefined}
+      rel={href ? 'noopener noreferrer' : undefined}
+      className={classes(
+        'relative rounded-lg p-4 text-center shadow-md',
+        !disabled &&
+          'transition-transform duration-[600ms] hover:scale-105 hover:bg-gray-50',
+        disabled && 'cursor-not-allowed bg-gray-200 opacity-50',
+      )}
+    >
+      <div
```

Comment:
I believe that SectionTile should be unaware of being wrapped with a link and in the places where you want it to be a link you can just wrap it with `<a ...>` by doing so we will avoid having weird anchors without links etc. and you won't need to have `disabled` state for anchor which in reality there's nothing like that in HTML.

Reply:
also this new SectionTile does not need to accept `disabled` prop, you can just inject `className` prop with specific styles for trading co-pilot